SeismoSocialDistancing.ipynb

```diff
@@ -325,7 +325,7 @@
     "    plt.gca().yaxis.set_major_formatter(ticks)\n",
     "    plt.ylabel(\"Displacement (nm)\")\n",
     "\n",
-    "    plt.title('Seismic Noise for %s - Filter: [%s] Hz' % (main,\n",
```

Comment:
Hey @claudiodsf , you spotted something I got wrong, I meant to do:
```
"    plt.title('Seismic Noise for %s - Filter: [%s] Hz' % (channelcode[-3:]+main,\n",
```
To it adapts to 3-components plots as well as single component...

Reply:
Ok, I'll fix it and force push

Reply:
Great ! Thanks